src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const items = document.querySelectorAll('li');
+
+function getNumber(el) {
+  return +el.dataset.salary.slice(1, el.length).split(',').join('.');
+}
+
+function sortList([...array], callback) {
+  return array.sort((a, b) => callback(b) - callback(a));
+}
+
+const sortedList = sortList(items, getNumber);
+
+const newList = document.getElementsByTagName('ul')[0];
```

Comment:
```suggestion
const newList = document.querySelector('ul');
```
in our case better to use querySelector, which return first found element instead collection, ad getElementByTagName